web/Dockerfile

```diff
@@ -3,32 +3,30 @@ FROM python:3.6
 WORKDIR /wwwroot
 
 COPY impact /wwwroot
-
 COPY scripts/start.sh /usr/bin
-
 COPY scripts/start-nodaemon.sh /usr/bin
 
 RUN apt-get update -y
 
 # python2.7 is required to run supervisor
-RUN apt-get install -y netcat mysql-client python2.7 python-setuptools nginx
+RUN apt-get install -y gettext netcat mysql-client nginx python2.7 python-setuptools
```

Comment:
Why the order change for `nginx`?  At first I thought it might be alphabetical, but then `netcat` is out of order.

Reply:
Was intended to be alphabetical. Fixed.